client/ayon_houdini/plugins/create/create_usd_componentbuilder.py

```diff
@@ -0,0 +1,69 @@
+import inspect
+
+from ayon_houdini.api import plugin
+from ayon_core.pipeline import CreatedInstance, CreatorError
+
+import hou
+
+
+class CreateUSDComponentBuilder(plugin.HoudiniCreator):
+    identifier = "io.ayon.creators.houdini.componentbuilder"
+    label = "USD Component Builder LOPs"
+    product_type = "usd"
+    icon = "cubes"
+    description = "Create USD from Component Builder LOPs"
+
+    def get_detail_description(self):
+        return inspect.cleandoc("""
+            Creates a USD publish from a Component Output LOP that is part of 
+            a solaris component builder network.
+            
+            The created USD will contain the component builder LOPs and all its
+            dependencies inside the single product.
+            
+            To use it, select a Component Output LOP and click "Create" for
+            this creator. It will generate an instance for each selected
+            Component Output LOP.
+        """)
+
+    def create(self, product_name, instance_data, pre_create_data):
+        nodes = hou.selectedNodes()
+        builders = [
+            node for node in nodes if node.type().name() == "componentoutput"
+        ]
```

Comment:
```suggestion
        builders = hou.lopNodeTypeCategory().nodeType("componentoutput").instances()
```

Reply:
Unfortunately this would return ALL nodes of them - and notes the ones in the `nodes` input list which is the user's selection. ;)

But likely doing a check like:

```python
if node.type() == hou.lopNodeTypeCategory().nodeType("componentoutput"):
```
Or something along those lines may make sense and is pretty readable/explicit.
Or maybe even easier, we just do `node.type().nameWithCategory()` and compare with the category included directly - I think I'll do that instead.

Reply:
```suggestion
        builders = [
            node for node in nodes if node.type().nameWithCategory() == "Lop/componentoutput"
        ]
```